api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -171,7 +171,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             new UserAuthentication(user, OAuth2Userinfo, token, UserType.RESEARCHER));
 
     // TODO: setup this in the context, get rid of log statement
-    log.log(Level.INFO, "{0} logged in", OAuth2Userinfo.getEmail());
+    log.log(Level.FINE, "{0} logged in", OAuth2Userinfo.getEmail());
```

Comment:
This was a very critical log line for debugging in GAE logs. The change is also entirely unrelated to the PR. Please revert this.

Reply:
OK. It's an annoyance locally and doesn't give me much context about what was going on. I thought it was just leftover and tried to clean it up.

Reply:
Is it supposed to fire every minute?

Reply:
No, though it might if you have local UI  tab open that's polling your API server. If you paste the logs you observed, I can try and answer. The log line is expected to run on every authenticated request.